zerver/logging_handlers.py

```diff
@@ -46,7 +47,10 @@ def add_request_metadata(report: Dict[str, Any], request: HttpRequest) -> None:
         else:
             user_full_name = user_profile.full_name
             user_email = user_profile.email
-            user_role = user_profile.get_role_name()
+            with override_language(user_profile.realm.default_language):
```

Comment:
We probably want to force English, since this goes only into an error-reporting email that goes only to server sysadmins, and all of our content for server sysadmins is in English.

Reply:
Hm, OK.  I didn't see any other overrides explicitly to english, and [`do_send_realm_reactivation_email` goes out to admins in the realm's default](https://github.com/zulip/zulip/blob/02725d32dd4945c1f17242371e7495db5a4809e4/zerver/lib/actions.py#L5978-L5988) -- but that's the realm admins, not the server admins.

Will switch to `settings.LANGUAGE_CODE`.